vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/Slide.java

```diff
@@ -127,12 +128,35 @@ public void setContents(List<IContentBlock> contents) {
     /*
      * (non-Javadoc)
      * 
-     * @see
-     * edu.asu.diging.vspace.core.model.impl.ISlide#setSequence(java.util.List)
+     * @see edu.asu.diging.vspace.core.model.impl.ISlide#setSequence(java.util.List)
      */
     public void setSequence(List<ISequence> sequence) {
         this.sequence = sequence;
     }
 
+    /**
+     * This Method will retrieve the first ImageBlock of a slide if the ImageBlock
+     * is present
+     * 
+     * @return ImageBlock
+     */
+    public ImageBlock getFirstImageBlock() {
+        List<IContentBlock> allBlocks = getContents();
+        if (allBlocks == null) {
```

Comment:
reverse the if statement (`if allBlocks != null`) and you'll cut down on the code.